spec/controllers/api/last_member_teams_controller_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe Api::LastMemberTeamsController do
+  include ControllerHelpers
+
+  let(:bob) { users(:bob) }
+  let(:alice) { users(:alice) }
+
+  context 'GET last_teammember_teams' do
+    it 'returns last teammember teams' do
+      login_as(:admin)
+
+      soloteam = Fabricate(:private_team)
+      user = soloteam.teammembers.first.user
+
+      get :index, params: { user_id: user.id }
+      team = JSON.parse(response.body)['data']['teams'][0]
```

Comment:
There's functions for this inside ControllerHelpers, at least once you rebase :)

Reply:
Yes, I even used it. :D 
But like with the storeStub I had some git issue. Didn't commit/push everything.